website/docs/commands/export.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Export ⚡️
+sidebar_position: 27
+---
+
+Export outputs information about current project to a format used by external build tools such as SBT or Mill.
+Additionally the command supports JSON format for custom analysis of projects.
+
+The project configuration is read both from information specified in source files
+as well as options passed to the `export` command
+
+Let's take a simple one-file project as an example:
+```scala title=Hello.scala
+//> using scala "3.1.3"
+//> using option "-Xasync"
+//> using dep "com.lihaoyi::os-lib:0.9.0"
+
+object Hello {
+  def main(args: Array[String]): Unit =
+    println(os.pwd)
+}
+```
+
+# Exporting to SBT:
+
+```bash ignore
+scala-cli export Hello.scala --sbt
+```
```

Comment:
````suggestion
```bash
scala-cli --power export Hello.scala --sbt
```
````
IMO totally worth it to run this on the CI